From 5f5bb7f350a728b89ee7b3fd0c4590738e7895d4 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Mon, 8 Oct 2018 19:20:22 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808-evb: fixup uart4_rts pinctrl
 error

Fixup the default pinctrl configuration for uart4-rts-gpio:
rockchip-pinctrl pinctrl: unable to find group for node uart4-rts-gpio

Change-Id: Ic25375aeba924ba0314ad5660b3ee4f18ea8c0cc
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index b2c1500ce639..94fba9e05526 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -650,8 +650,10 @@
 		};
 	};
 
-	uart4_rts_gpio: uart4-rts-gpio {
-		rockchip,pins = <4 RK_PB7 0 &pcfg_pull_none>;
+	wireless-bluetooth {
+		uart4_rts_gpio: uart4-rts-gpio {
+			rockchip,pins = <4 RK_PB7 0 &pcfg_pull_none>;
+		};
 	};
 
 	wireless-wlan {
-- 
2.35.3

